app/views/clients/clients.html

```diff
@@ -30,8 +30,9 @@
         </tr>
         </tbody>
     </table>
-    <ul class="pager">
+    <!-- <ul class="pager">
         <li class="previous"><a id="prev" ng-click="clients.previous()" href="" ng-disabled="!clients.hasPrevious()">&larr; {{'label.button.previous' | translate}}</a></li>
         <li class="next"><a id="next" ng-click="clients.next()" href="" ng-disabled="!clients.hasNext()">{{'label.button.next' | translate}} &rarr;</a></li>
-    </ul>
+    </ul> -->
+    <dir-pagination-controls template-url="../../bower_components/angular-utils-pagination/dirPagination.tpl.html" on-page-change="getResultsPage(newPageNumber)"></dir-pagination-controls>
```

Comment:
Also, access template-url="../../bower_components/angular-utils-pagination/dirPagination.tpl.html" from the context of index.html, i,e  template-url="bower_components/angular-utils-pagination/dirPagination.tpl.html" , this would ensure that the relative files are found not only when you host the community-app on a web server (i.e grunt server), but directly access index.html from the local file system too....